config/schema/performance_budget.schema.yml

```diff
@@ -10,3 +10,50 @@ performance_budget.performance_budget.*:
       label: 'Label'
     uuid:
       type: string
+    cron_schedule:
+      type: string
+      label: 'Cron Schedule'
+    assessment_utilities:
+      type: sequence
+      sequence:
+        type: mapping
+        mapping:
+          uuid:
+            type: string
+          id:
+            type: string
+    assessments:
+      type: sequence
+      sequence:
+        type: mapping
+        mapping:
+          uuid:
+            type: string
+          status:
+            type: string
+            label: 'Status'
+          timestamp:
+            type: string
+            label: 'Timestamp'
+          queue_ct:
+            type: integer
+            label: 'Queue Ct'
+          captures:
+            type: sequence
+            sequence:
+              type: mapping
+              mapping:
+                uuid:
+                  type: string
+                status:
+                  type: string
+                  label: 'Status'
+                timestamp:
+                  type: string
+                  label: 'Timestamp'
+                capture_type:
+                  type: string
+                  label: 'Capture Type'
+                content:
+                  type: string
+                  label: 'Capture Content'
```

Comment:
Irrelevant schema fields here. I'm assuming this will be modified at a future point. Just pointing it out.